framework/source/class/qx/core/MObjectId.js

```diff
@@ -63,16 +64,26 @@ qx.Mixin.define("qx.core.MObjectId", {
     __changingOwner: false,
 
     /**
-     * apply owner id
+     * Apply owner
      */
     _applyOwner : function(value, oldValue) {
       if (!this.__changingOwner) {
         throw new Error("Please use API methods to change owner, not the property");
       }
     },
+    
+    /**
+     * Transform/validate objectId
+     */
+    _transformObjectId: function(value) {
```

Comment:
This method isn't actually doing any transforms. Would it not be cleaner to have a customer `check` function that checked for string and no slashes, than to do it in a non-transforming `transform` function?

Reply:
yes i agree - funny, i've never used that feature before!